src/app/EventManagement.h

```diff
@@ -189,6 +189,18 @@ struct LogStorageResources
         PriorityLevel::Invalid; // Log priority level associated with the resources provided in this structure.
 };
 
+/**
+ * @brief
+ *  A EventScheduler to schedule deliveries of events.
```

Comment:
Can we describe this somehow else to help read the code? It seems obvious that a class named `Event scheduler` is used to `schedule events`.

Maybe some background of intent would be useful.

Reply:
Separate EventScheduler to its own header file solves the problem